README.md

```diff
@@ -0,0 +1,94 @@
+Ecosystem Edge NVIDIA-CI - Golang Automation CI
+=======
+# NVIDIA-CI
+
+## Overview
+The [nvidia-ci](https://github.com/rh-ecosystem-edge/nvidia-ci) is an automation/CI framework to test NVIDIA operators, starting with the GPU Operator.
+This project is based on golang + [ginkgo](https://onsi.github.io/ginkgo) framework.
+
+### Project requirements
+* golang v1.20.x
+* ginkgo v2.x
+
+## nvidia-ci
```

Comment:
Should we move that entire section [here](https://github.com/rh-ecosystem-edge/nvidia-ci/pull/11/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R5)?

Reply:
Updated